crates/turborepo-lib/src/commands/login.rs

```diff
@@ -5,29 +5,39 @@ use turborepo_auth::{login as auth_login, sso_login as auth_sso_login};
 use crate::commands::CommandBase;
 
 pub async fn sso_login(base: &mut CommandBase, sso_team: &str) -> Result<()> {
-    let api_client: APIClient = base.api_client()?;
     let ui = base.ui;
+    let api_client: APIClient = base.api_client()?;
     let login_url_config = base.repo_config()?.login_url().to_string();
+    let token_path = base.repo_config()?.get_path().to_owned();
 
     // We are passing a closure here, but it would be cleaner if we made a
     // turborepo-config crate and imported that into turborepo-auth.
     let set_token = |token: &str| -> Result<(), anyhow::Error> {
         Ok(base.user_config_mut()?.set_token(Some(token.to_string()))?)
     };
 
-    auth_sso_login(api_client, &ui, set_token, &login_url_config, sso_team).await
+    auth_sso_login(
+        api_client,
+        &ui,
+        &token_path,
+        set_token,
+        &login_url_config,
+        sso_team,
+    )
+    .await
 }
 
 pub async fn login(base: &mut CommandBase) -> Result<()> {
     let api_client: APIClient = base.api_client()?;
     let ui = base.ui;
     let login_url_config = base.repo_config()?.login_url().to_string();
+    let token_path = base.repo_config()?.get_path().to_owned();
 
     // We are passing a closure here, but it would be cleaner if we made a
     // turborepo-config crate and imported that into turborepo-auth.
     let set_token = |token: &str| -> Result<(), anyhow::Error> {
         Ok(base.user_config_mut()?.set_token(Some(token.to_string()))?)
```

Comment:
Use `token_path` here too (or go all the way and remove this closure and use `token_path` in the crate)

Reply:
We talked in a call about this - there will be a follow up PR with cleanup efforts around this.

Reply:
missed a comment here, but totally nonblocking, you and me are WELL AWARE of this quirk and have talked about it enough 😅